crates/next-custom-transforms/src/transforms/fonts/mod.rs

```diff
@@ -74,14 +74,28 @@ impl VisitMut for NextFontLoaders {
                 };
             items.visit_with(&mut wrong_scope);
 
-            // Remove marked module items
-            items.retain(|item| !self.state.removeable_module_items.contains(&item.span_lo()));
+            fn is_removable(ctx: &NextFontLoaders, item: &ModuleItem) -> bool {
+                ctx.state.removeable_module_items.contains(&item.span_lo())
+            }
+
+            let first_removable_index = items
+                .iter()
+                .position(|item| is_removable(self, item))
+                .unwrap();
 
-            // Add font imports and exports
             let mut new_items = Vec::new();
```

Comment:
Why do we even need `new_items`? It should be equivalent to this which also does less moving around
```rust
            // Remove marked module items
            items.retain(|item| !is_removable(self, item));

            // Add font imports and exports
            items.splice(
                first_removable_index..first_removable_index,
                std::mem::take(&mut self.state.font_imports),
            );
            items.append(&mut self.state.font_exports);
```

Reply:
oh yeah you're right! (didn't question it bc that's how it was before but it doesn't rly make sense)